### ISSUE TITLE
Use `scylla_manager_repair_progress` metric from Scylla Manager on SM dashboard

### ISSUE THREAD

**Issue by karol-kokoszka** [scylladb/scylla-monitoring#2122]
**Describe the feature and the current behavior/state.**

Right now, monitoring is recording the repair progress metric on its own.
https://github.com/scylladb/scylla-monitoring/blob/22d8d0c187884f00e21c6d59ce9e83b7b456ceb4/prometheus/prom_rules/prometheus.latency.rules.yml#L46C1-L47
... and the metric is shown on the following dashboard 
https://github.com/scylladb/scylla-monitoring/blob/22d8d0c187884f00e21c6d59ce9e83b7b456ceb4/grafana/scylla-manager.template.json#L129-L164

With the Scylla Manager 3.2.2 we introduced another metric that is tracking the total progress of the repair.
It's `scylla_manager_repair_progress`.
It was introduced with the following PR https://github.com/scylladb/scylla-manager/pull/3566

The metrics is labelled just with the `cluster` which is the cluster-id of where the repair was performed.

**Who will benefit with this feature?**

https://github.com/scylladb/scylla-enterprise/issues/3582

**Any Other info.**

**Comment by amnonh** [comment]
There a minor issue, the monitor dashboard don't support different patch releases, the question, when will we have a minor release with this change? I would like to make it part of 4.6, but have no clean way of doing that

**Comment by amnonh** [comment]
@mykaul we need to address the the issue in order to solve the issue. When is the next manager minor release expected?

**Comment by mykaul** [comment]
@karol-kokoszka ^^^ ?

**Comment by karol-kokoszka** [comment]
@mykaul @amnonh We are targeting 3.3.0 as the next minor release, where we want to support small_tables_optimization repair api and give possibility to extend parallelism on repairs. But we have no ETA defined yet.

**Comment by mykaul** [comment]
@karol-kokoszka , @Michal-Leszczynski - any updates?

**Comment by Michal-Leszczynski** [comment]
The `scylla_manager_repair_progress` metric is present in since SM 3.2.3.

**Comment by tarzanek** [comment]
ping @amnonh 
3.2.5 is out and I see above creating problems (sometimes we see weird values)
e.g. 
![image](https://github.com/scylladb/scylla-monitoring/assets/504773/92c29907-3e38-4b04-93db-7974624c8d30)

but monitor shows:
![image](https://github.com/scylladb/scylla-monitoring/assets/504773/56ecac3b-e792-4c36-927f-158de28d7560)


while real number is really 38%:
```
 sctool progress repair/ZZZ -c XXXX  | head
Run:            c25158f4-c5ce-11ee-9f9e-027ab8c91b0b
Status:         RUNNING
Start time:     07 Feb 24 15:37:10 UTC
Duration:       3m33s
Progress:       38%
Intensity:      1/6
Parallel:       12/12 (max)
Datacenters:
  - AWS_EU_WEST_1
```

and just for completness the internally counted value which is wrong:
```
![image](https://github.com/scylladb/scylla-monitoring/assets/504773/b1091202-3fe6-4ff0-97f5-254f535e03d1)

```

**Comment by tarzanek** [comment]
$ sctool version
Client version: 3.2.5-0.20231206.8b378dea
Server version: 3.2.5-0.20231206.8b378dea

the completness pic one more:
![image](https://github.com/scylladb/scylla-monitoring/assets/504773/f6c56b2c-0b34-449f-9326-f45402e109b5)

**Comment by amnonh** [comment]
@tarzanek, the issue remains: the manager updates metrics on a patch release; we don't support different dashboards per patch release, only on minor releases.
When will 3.3 is going to be released?

**Comment by tarzanek** [comment]
can we mandate SM and SM agent upgrade with monitor version?

e.g. release 4.7 that only supports SM 3.2.5 ?

**Comment by amnonh** [comment]
@tarzanek I don't think I can add that dependency for the cloud. I can release a 3.3 dashboard with those metrics so they can choose what to start.
In general, it's not a very good practice

**Comment by amnonh** [comment]
@tarzanek I'll make an adjustment to support both options. Can you help me with a test that show the new metric in action?

**Comment by amnonh** [comment]
Fixed by #2169
